fs/expose/dokan/__init__.py

```diff
@@ -440,29 +461,8 @@ def CreateFile(self, path, access, sharing, disposition, flags, info):
 
     @timeout_protect
     @handle_fs_errors
-    def OpenDirectory(self, path, info):
-        path = normpath(path)
-        if self._is_pending_delete(path):
-            raise ResourceNotFoundError(path)
-        if not self.fs.isdir(path):
-            if not self.fs.exists(path):
-                raise ResourceNotFoundError(path)
-            else:
-                raise ResourceInvalidError(path)
-        info.contents.IsDirectory = True
-
-    @timeout_protect
-    @handle_fs_errors
-    def CreateDirectory(self, path, info):
-        path = normpath(path)
-        if self._is_pending_delete(path):
-            return -ERROR_ACCESS_DENIED
-        self.fs.makedir(path)
-        info.contents.IsDirectory = True
-
-    @timeout_protect
-    @handle_fs_errors
     def Cleanup(self, path, info):
+        path = path.replace('\\', '/')
```

Comment:
See https://github.com/PyFilesystem/pyfilesystem/issues/256#issuecomment-236356489
I found no other wait to make it work.
Dokan is only Windows FS.